packages/cosmos_ui_components/lib/components/template/cosmos_wallet_list_item.dart

```diff
@@ -7,47 +8,54 @@ class CosmosWalletListItem extends StatelessWidget {
   final String address;
   final bool isSelected;
   final VoidCallback onClicked;
+  final Widget? suffix;
 
   const CosmosWalletListItem({
     Key? key,
     required this.name,
     required this.address,
     required this.isSelected,
     required this.onClicked,
+    this.suffix,
   }) : super(key: key);
 
   @override
   Widget build(BuildContext context) {
+    final theme = CosmosTheme.of(context);
     return InkWell(
       onTap: onClicked,
-      child: Container(
-        padding: EdgeInsets.all(CosmosTheme.of(context).spacingL),
-        margin: EdgeInsets.symmetric(
-          horizontal: CosmosTheme.of(context).spacingL,
-          vertical: CosmosTheme.of(context).spacingM,
-        ),
-        decoration: BoxDecoration(
-          borderRadius: CosmosTheme.of(context).borderRadiusM,
-          // TODO: This color needs to be fixed. Doesn't match the dark theme.
-          border: Border.all(color: isSelected ? Colors.black12 : Colors.transparent),
-        ),
-        child: Padding(
-          padding: EdgeInsets.symmetric(vertical: CosmosTheme.of(context).spacingM),
-          child: Row(
-            mainAxisAlignment: MainAxisAlignment.spaceBetween,
-            children: [
-              Row(
+      child: Column(
+        children: [
+          Container(
+            padding: EdgeInsets.all(theme.spacingL),
+            margin: EdgeInsets.symmetric(
+              horizontal: theme.spacingL,
+              vertical: theme.spacingM,
+            ),
+            decoration: BoxDecoration(
+              borderRadius: theme.borderRadiusM,
+              // TODO: This color needs to be fixed. Doesn't match the dark theme.
+              border: Border.all(color: isSelected ? Colors.black12 : Colors.transparent),
```

Comment:
we should use theme here for the colors. this way we will automatically start supporting dark theme when we introduce separate theme values for dark mode

Reply:
Updated